src/test/java/com/epam/tc/hw7/MetalsPageTestData.java

```diff
@@ -0,0 +1,14 @@
+package com.epam.tc.hw7;
+
+import com.jdiai.tools.DataClass;
+import java.util.List;
+
+public class MetalsPageTestData extends DataClass {
```

Comment:
The DateClass is parameterized
![image](https://user-images.githubusercontent.com/74856431/216546888-821b4288-3afb-446c-a03b-ead78de9ee7d.png)